client/examples/cli/main.rs

```diff
@@ -291,14 +289,14 @@ async fn command_generate_bip39(passphrase: Option<String>, language: MnemonicLa
         stronghold::Location::generic(vault_path.as_bytes().to_vec(), record_path.as_bytes().to_vec());
 
     let bip39_procedure = BIP39Generate {
-        passphrase,
+        passphrase: passphrase.unwrap_or("".to_owned()).into(),
```

Comment:
For here and other instances, maybe using `unwrap_or_default` is a bit more idiomatic

Reply:
Done.